### PR TITLE
Simplify a check

### PR DESCRIPTION
**Pull request by GuilhemN**

### CODE REVIEW

File: Security/Voter/IsOwnerVoter.php [hunk 1]
```diff
@@ -20,13 +20,7 @@ public function supportsAttribute($attribute)
...
     public function supportsClass($class)
     {
-        if (is_object($class)) {
-            $refl = new \ReflectionObject($class);
-
-            return $refl->implementsInterface('Knp\RadBundle\Security\OwnableInterface');
-        }
-
-        return false;
+        return is_subclass_of($class, 'Knp\RadBundle\Security\OwnableInterface');
```
Comment (apfelbox) [inline Security/Voter/IsOwnerVoter.php]:
``` php
return $class instanceof Knp\RadBundle\Security\OwnableInterface;
```

might be better readable?

Comment (apfelbox) [inline Security/Voter/IsOwnerVoter.php]: The new method will accept a string as value (like a class name implementing `OwnableInterface`, whereas the old one did not.

Comment (GuilhemN) [reply Security/Voter/IsOwnerVoter.php]: You're right :+1: I didn't realize an object was used here

**Comment by einenlum** [comment]
This bundle is not maintained anymore.

**Pull request closed without merging**